bitcoind/getrawtransaction.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: ISC
+// Copyright (c) 2019-2020 Bitmark Inc.
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package bitcoind
+
+import (
+	"encoding/json"
+)
+
+type Vin struct {
+	TxID string `json:"txid"`
+	Vout int64  `json:"vout"`
+}
+
+type Vout struct {
+	Value        float64 `json:"value"`
+	N            int     `json:"n"`
+	ScriptPubKey struct {
+		Asm       string   `json:"asm"`
+		Hex       string   `json:"hex"`
+		ReqSigs   int      `json:"reqSigs"`
+		Type      string   `json:"type"`
+		Addresses []string `json:"addresses"`
+	} `json:"scriptPubKey,omitempty"`
+}
+type RawTransaction struct {
+	TxID          string `json:"txid"`
+	Hash          string `json:"hash"`
+	Version       int    `json:"version"`
+	Size          int    `json:"size"`
+	Vsize         int    `json:"vsize"`
+	Weight        int    `json:"weight"`
+	Locktime      int    `json:"locktime"`
+	Vins          []Vin  `json:"vin"`
+	Vouts         []Vout `json:"vout"`
+	Hex           string `json:"hex"`
+	Blockhash     string `json:"blockhash"`
+	Confirmations int    `json:"confirmations"`
+	Time          int    `json:"time"`
+	Blocktime     int    `json:"blocktime"`
+}
+
+func (c *HttpBitcoind) GetRawTransaction(txId string) (*RawTransaction, error) {
```

Comment:
Could be done by the client from `NewBtcdRPCClient`.

Reply:
> Add the license for `bitcoind/bitcoind.go`.
> 
> For explicit Bitcoin RPC calls in the pod controller, I'd suggest using the client from btcd to make it simpler. I don't strongly request changes on these.

I view the btcd client, and there are some param missing in GetTransaction, so I have to use our own client to call the rpc with correct params.